File: clear_books.py
import os
from supabase import create_client
from dotenv import load_dotenv

load_dotenv()
url = os.environ.get('SUPABASE_URL')
key = os.environ.get('SUPABASE_KEY')

if not url or not key:
    print("No Supabase credentials found")
    exit(1)

client = create_client(url, key)

try:
    res = client.table('books').delete().neq('id', 0).execute()
    print("Cleared books:", res.data)
except Exception as e:
    print("Error:", e)

File: frontend/clear_books.js
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';
dotenv.config({ path: '../.env' });

const url = process.env.VITE_SUPABASE_URL;
const key = process.env.VITE_SUPABASE_ANON_KEY;
const supabase = createClient(url, key);

async function clear() {
    console.log("Clearing books...");
    const { data, error } = await supabase.from('books').delete().neq('id', 0);
    if (error) console.error(error);
    else console.log("Cleared successfully", data);
}
clear();
